src/NUnitTestAdapter/NUnit3TestDiscoverer.cs

```diff
@@ -84,7 +84,7 @@ public void DiscoverTests(IEnumerable<string> sources, IDiscoveryContext discove
                 }
                 catch (TypeLoadException ex)
                 {
-                    if (ex.TypeName == NUnit3FrameworkDriver.CONTROLLER_TYPE)
+                    if (ex.TypeName == "NUnit.Framework.Api.FrameworkController")
```

Comment:
I am not sure about this change.  Now you have that explicit string both there and in the NUnit3FrameworkDriver class, and they need to match.